Projects/App/Sources/Global/Extension/UIDevice+.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  UIDevice+.swift
+//  App
+//
+//  Created by Chanhee Jeong on 2022/10/25.
+//  Copyright © 2022 zesty. All rights reserved.
+//
+
+import UIKit
+
+public extension UIDevice {
+    var hasNotch: Bool {
+        if #available(iOS 11.0, *) {
+            let bottom = UIApplication.shared.keyWindow?.safeAreaInsets.bottom ?? 0
```

Comment:
p2
keyWindow deprecated 됐네요. 아래와 같이 쓰면 되나봅니다
```suggestion
            let keyWindow = UIApplication.shared.connectedScenes
                    .filter({$0.activationState == .foregroundActive})
                    .compactMap({$0 as? UIWindowScene})
                    .first?.windows
                    .filter({$0.isKeyWindow}).first
            let bottom = UIApplication.shared.keyWindow?.safeAreaInsets.bottom ?? 0
```
[deprecated in iOS 13.0](https://stackoverflow.com/questions/57134259/how-to-resolve-keywindow-was-deprecated-in-ios-13-0)

Reply:
![good](https://user-images.githubusercontent.com/63157395/197678105-77d4b545-54d2-404a-afc2-73468dadf560.png)

Reply:
a11048b52347cbf79d7ac7fd4d9c723f97516a4a